docs/development/pigs/pig-009.rst

```diff
@@ -0,0 +1,231 @@
+.. include:: ../../references.txt
+
+.. _pig-009:
+
+**********************
+PIG 9 - Event Sampling
+**********************
+
+* Author: Fabio Pintore, Andrea Giuliani, Axel Donath
+* Created: May 03, 2019
+* Accepted: 
+* Status: Draft
+* Discussion: `PR 2136`_
```

Comment:
Please, for consistency, use "GH" instead of "PR", throughout this PIG.

Example:  https://raw.githubusercontent.com/gammapy/gammapy/master/docs/development/pigs/pig-001.rst